Add interfaces/mathml-core.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/mathml-core.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389

stephenmcgruer commented:
This will need an equivalent idlharness test

foolip commented on interfaces/mathml-core.idl:
```diff
@@ -0,0 +1,28 @@
+interface MathMLElement : Element { };
+MathMLElement includes GlobalEventHandlers;
+MathMLElement includes DocumentAndElementEventHandlers;
+MathMLElement includes HTMLOrForeignElement;
```
I tried adding a test for this, but it's blocked on the `HTMLOrForeignElement` mixin not existing. That's blocked on https://github.com/whatwg/html/issues/4702.

autofoolip committed:
Add interfaces/mathml-core.idl

Source: https://github.com/w3c/webref/blob/0cb115a/ed/idl/mathml-core.idl
Build: https://travis-ci.org/w3c/webref/builds/723814389
